index.html

```diff
@@ -16,5 +16,36 @@
             </div>
         </nav>
     </header>
+    <main id="content">
+        <section id="container-1">
+            <h1 id="headline">Hey There.<br>I'm Amrit</h1>
+            <h2 id="title-line">I'am a Software Developer</h2>
+            <p id="para">I can help you build a product, feature or website Look through some of my work <br> and experience! If you like what you see <br> and have a project you need coded, 
+                don't hestiate to contact me.</p>
+        </section>
+        <section id="container-2">
+            <div id="socials-container">
+                <div class="socials">
+                    <img src="images/github.png" alt="github link" class="social-img">
+                </div>
+                <div class="socials">
+                    <img src="images/linkedin.png" alt="linkedin link" class="social-img">
+                </div>
+                <div class="socials">
+                    <img src="images/dnt.png" alt="dnt link" class="social-img">
+                </div>
+                <div class="socials">
+                    <img src="images/tweet.png" alt="Twitter link" class="social-img">
+                </div>
+                <div class="socials">
+                    <img src="images/microverse.png" alt="Microverse link" class="social-img">
+                </div>
+                <div id="bg-container">
+                    <img id="bg-img" src="images/Groupbackground.png" alt="Background card">
+                </div>   
+            </div>
+             
+        </section>
```

Comment:
- [ ] It is required to use `ul` to wrap all your social icon, use `li` to list each of them and wrap each of the icons in a link (i.e `a` tag). You don't need to declare a separate `section` for the social icons, they should be inside the section you've already declared.